tasmota/xdrv_63_robotdyn.ino

```diff
@@ -0,0 +1,253 @@
+/*
+  xdrv_63_robotdyn.ino -support for robotdin 4 AC channel dimmer with zero crossing
+
+  SPDX-FileCopyrightText: 2022 Benoit Lecuppe
+
+  SPDX-License-Identifier: GPL-3.0-only
+*/
+//ifdef VSCODE_DEV
+  #define USE_LIGHT
+  #define USE_ROBOTDYN_DIMMER
```

Comment:
This #define should go in my_user_config.h file and it should go commented out. Please, move it to my_user_config.h as:

```cpp
// #define USE_ROBOTDYN_DIMMER
```